### ISSUE TITLE
Doesn't find Chrome binary on mac

### ISSUE THREAD

**Issue by marvinhagemeister** [GoogleChrome/lighthouse#827]
Just intalled lighthouse via:

```bash
npm i -g lighthouse
```

But when I try to run it, lighthouse is not able to find a Chrome binary. I'm running Chrome v54 on my machine.

error log:
````bash
marvinhagemeister@MBPro:~ $ lighthouse --select-chrome /Applications/Google\ Chrome.app
Launching Chrome...
created /var/folders/g7/4ch_02hj01lgx536vxs_nbpm0000gn/T/lighthouse.XXXXXXX.sez6VQOG
Runtime error encountered: Error: No Chrome Installations Found
    at Promise.resolve.then (/usr/local/lib/node_modules/lighthouse/lighthouse-cli/chrome-launcher.js:94:33)
    at process._tickCallback (internal/process/next_tick.js:103:7)
Error: No Chrome Installations Found
    at Promise.resolve.then (/usr/local/lib/node_modules/lighthouse/lighthouse-cli/chrome-launcher.js:94:33)
    at process._tickCallback (internal/process/next_tick.js:103:7)
marvinhagemeister@MBPro:~ $ 
```

**Comment by brendankenny** [comment]
`--select-chrome` is a boolean option, to indicate whether you want to interactively select which Chrome to run if Lighthouse finds multiple versions on your machine. If you don't include that flag, Lighthouse will just pick one, prioritizing Canary and versions found under `/Applications/`. I don't think we have a way to tell LH a direct path for Chrome on a Mac, though we probably could/should.

It's possible `--select-chrome` is a bad choice of name if it's confusing in that way. Maybe we could pick something better.

**Comment by marvinhagemeister** [comment]
@brendankenny That's a great suggestion. I was unsure if that was a typo in the cli help.

After a little mit more debugging I found the issue. On mac lighthouse specifically searches for `google chrome canary` which I had not installed instead of the normal chrome binary.

**Comment by brendankenny** [comment]
> On mac lighthouse specifically searches for google chrome canary which I had not installed instead of the normal chrome binary

Got it. FWIW, this was fixed in #782, which will be in the next release (coming shortly)

**Comment by marvinhagemeister** [comment]
@brendankenny Awesome! Thank you so much 👍
